src/java.ts

```diff
@@ -101,15 +101,20 @@ export async function getJavaConfig(
         return javaConfigDisabledInLightWeightMode();
       }
       const isTest: boolean = await extensionApi.isTestFile(fileUri);
-      const sourceLevel: string = (
-        await extensionApi.getProjectSettings(fileUri, ['org.eclipse.jdt.core.compiler.compliance'])
-      )['org.eclipse.jdt.core.compiler.compliance'];
+      const projectSettings: { [name: string]: string } = await extensionApi.getProjectSettings(fileUri, [
+        'org.eclipse.jdt.core.compiler.compliance',
+        'org.eclipse.jdt.ls.core.vm.location'
```

Comment:
Nitpick: maybe extract these 2 strings to constants

Reply:
:+1: